inspec.yml

```diff
@@ -0,0 +1,611 @@
+name: cms-ars-5.0-red-hat-enterprise-linux-8-stig-overlay
+title: cms-ars-5.0-red-hat-enterprise-linux-8-stig-overlay
+maintainer: CMS SAF team
+copyright: .
+copyright_email: saf@cms.hhs.gov
+license: Apache-2.0
+summary: "CMS ARS 5.0 Overlay InSpec Validation Profile for RHEL8 STIG"
+version: 1.3.0
+inspec_version: ">= 4.0"
+
+depends:
+- name: redhat-enterprise-linux-8-stig-baseline
+  url: https://github.com/ssayed118/redhat-enterprise-linux-8-stig-baseline/archive/vagrant.tar.gz
```

Comment:
Before the PR is merged, the branch name needs to be updated to `main`

Reply:
Done https://github.com/CMSgov/cms-ars-5.0-red-hat-enterprise-linux-8-stig-overlay/pull/1/commits/c41e4e27b571f01c51487bd1b649a611721df682